src/Mod/AddonManager/addonmanager_firstrun.py

```diff
@@ -51,55 +52,28 @@ def exec(self) -> bool:
-
-            warning_dialog.checkBoxAutoCheck.setChecked(autocheck)
-            warning_dialog.checkBoxDownloadMacroMetadata.setChecked(download_macros)
-            warning_dialog.comboBoxProxy.setCurrentIndex(proxy_option)
-            toggle_proxy_list(proxy_option)
-            if proxy_option == 2:
-                warning_dialog.lineEditProxy.setText(proxy_string)
-
-            warning_dialog.comboBoxProxy.currentIndexChanged.connect(toggle_proxy_list)
-
-            warning_dialog.labelWarning.setStyleSheet(
-                f"color:{utils.warning_color_string()};font-weight:bold;"
-            )
 
+            # Set up the warning icon
+            warning_icon_label = warning_dialog.warningIconLabel
+            pixmap = QPixmap(":/icons/AddonMgrWithWarning.svg")
+            warning_icon_label.setPixmap(pixmap)
+            warning_icon_label.setScaledContents(True)
+            warning_icon_label.setFixedSize(80, 80)
+
+            # Customize layout margins and spacing
+            layout = warning_dialog.findChild(QtWidgets.QHBoxLayout, "bodyLayout")
+            layout.setContentsMargins(20, 6, 20, 6)  # Left, top, right, bottom margins
+            layout.setSpacing(30)  # Set spacing between widgets
+
+            # Connect the "Continue" button to accept the dialog (continue with Addon Manager)
+            warning_dialog.buttonContinue.clicked.connect(warning_dialog.accept)
+
+            # Connect the "Quit" button to reject (exit) the dialog
+            warning_dialog.buttonQuit.clicked.connect(warning_dialog.reject)
+
+            # Show the dialog and check whether the user accepted or canceled
             if warning_dialog.exec() == QtWidgets.QDialog.Accepted:
                 self.readWarning = True
                 self.pref.SetBool("readWarning2022", True)
-                self.pref.SetBool("AutoCheck", warning_dialog.checkBoxAutoCheck.isChecked())
-                self.pref.SetBool(
-                    "DownloadMacros",
-                    warning_dialog.checkBoxDownloadMacroMetadata.isChecked(),
-                )
-                selected_proxy_option = warning_dialog.comboBoxProxy.currentIndex()
-                if selected_proxy_option == 0:
-                    self.pref.SetBool("NoProxyCheck", True)
-                    self.pref.SetBool("SystemProxyCheck", False)
-                    self.pref.SetBool("UserProxyCheck", False)
-                elif selected_proxy_option == 1:
-                    self.pref.SetBool("NoProxyCheck", False)
-                    self.pref.SetBool("SystemProxyCheck", True)
-                    self.pref.SetBool("UserProxyCheck", False)
-                else:
-                    self.pref.SetBool("NoProxyCheck", False)
-                    self.pref.SetBool("SystemProxyCheck", False)
-                    self.pref.SetBool("UserProxyCheck", True)
-                    self.pref.SetString("ProxyUrl", warning_dialog.lineEditProxy.text())
```

Comment:
The previous version had three options:
* auto check addons for updates
* download macros
* proxy settings

In the new version none of these are available and will default to:
* [auto check = False](https://github.com/FreeCAD/FreeCAD/blob/main/src/Mod/AddonManager/AddonManager.py#L274)
* [download macros = False](https://github.com/FreeCAD/FreeCAD/blob/main/src/Mod/AddonManager/AddonManager.py#L480)
* proxy settings = No proxy check.

Are these settings what we want for users?
I would assume that NoProxyCheck + AutoCheck + DownloadMacros would be best for most.

It would be great if someone else could weigh in on this.

Reply:
I agree -- if we aren't going to present the options to the user, then the defaults should be as you suggest. I had those defaults set like that predicated on the user being presented the options on first run, I did expect that most people would opt for the slower (but more useful) options.

Reply:
As always, if you need help with it, hit us up on Discord. I should be around after the PR Review Meeting today.

Reply:
It'll be fine! I can help you out if you get into problems.
You can push the changes to the same branch and this PR will update automatically (no need to create a new PR).

Reply:
Comments incorporated. Please let me know if there is anything that needs amending.